libraries/src/Cache/Storage/RedisStorage.php

```diff
@@ -25,6 +25,16 @@
  */
 class RedisStorage extends CacheStorage
 {
+    /**
+     * Cache key 
```

Comment:
```suggestion
     * Cache key
```
Fix PHPCS (there should be no trailing whitespace in comments).

Reply:
👍